torch/_subclasses/fake_tensor.py

```diff
@@ -200,8 +200,10 @@ def maybe_get_fake_mode(t):
     if isinstance(t, FakeTensor):
         return t.fake_mode
     if is_traceable_wrapper_subclass(t):
-        inner_tensors, _ = t.__tensor_flatten__()
-        modes = [maybe_get_fake_mode(x) for x in inner_tensors]
```

Comment:
this always returns None because it's checking the string attribute names, not the attributes

Reply:
darn, this was my fault - thanks for fixing @vkuzo!